web_programming/fetch_github_info.py

```diff
@@ -8,19 +9,23 @@
 for more information around suggested workarounds and removal dates.
 """
 
+"""
+Created by lawric1 on 24/11/20
```

Comment:
This was not a creation of an algorithm but just an update. Please remove this or convert this into an `Update by` field.